app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,46 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [ :show, :edit, :update, :destroy ]
```

Comment:
この書き方がコード少なくて見やすいので合わせてほしいです。
```suggestion
  before_action :set_post, only: %i[show edit update destroy]
```